File: trunk/ blackjack-tamu-431/players/Playerable.java
package players;

import engine.Card;

public interface Playerable
{
   public void addCardToHand( Card temp );

   public void resetHand();

   public boolean isHitting();

   public void setWinCount( int numWins );

   public int getWinCount();

   public int getHandSize();

   public int getHandValue();
}

File:  blackjack-tamu-431/AY/BlackJack-CS/BlackJack-CS/Playerable.cs
using engine;

namespace player {

public interface Playerable
{
   void addCardToHand( Card temp );

   void resetHand();

   bool isHitting();

   void setWinCount( int numWins );

   int getWinCount();

   int getHandSize();

   int getHandValue();
}
}